src/zql/ivm/graph/difference-stream.ts

```diff
@@ -43,48 +49,83 @@ import {QueueEntry} from './queue.js';
  */
 // T extends object: I believe in the context of ZQL we only deal with object.
 export class DifferenceStream<T extends object> {
```

Comment:
Do we want to use extends Entity here?

Reply:
I think so. I wanted to confirm with you or @grgbkr that we can always expect all queryable things to have an `id` in Replicache.